ui/StatusQ/src/StatusQ/Components/StatusMessage.qml

```diff
@@ -121,299 +121,300 @@ Rectangle {
-    }
 
-    Rectangle {
-        anchors {
-            top: parent.top
-            bottom: parent.bottom
-            left: parent.left
+        Rectangle {
+            anchors {
+                top: parent.top
+                bottom: parent.bottom
+                left: parent.left
+            }
+            width: 2
+            visible: root.hasMention
+            color: Theme.palette.mentionColor1
         }
-        width: 2
-        visible: root.hasMention
-        color: Theme.palette.mentionColor1
-    }
 
-    SequentialAnimation {
-        id: messageFoundAnimation
+        SequentialAnimation {
+            id: messageFoundAnimation
 
-        PauseAnimation {
-            duration: 600
-        }
-        NumberAnimation {
-            target: highlightRect
-            property: "opacity"
-            to: 1.0
-            duration: 1500
-        }
-        PauseAnimation {
-            duration: 1000
-        }
-        NumberAnimation {
-            target: highlightRect
-            property: "opacity"
-            to: 0.0
-            duration: 1500
+            PauseAnimation {
+                duration: 600
+            }
+            NumberAnimation {
+                target: highlightRect
+                property: "opacity"
+                to: 1.0
+                duration: 1500
+            }
+            PauseAnimation {
+                duration: 1000
+            }
+            NumberAnimation {
+                target: highlightRect
+                property: "opacity"
+                to: 0.0
+                duration: 1500
+            }
         }
-    }
 
-    Rectangle {
-        id: highlightRect
-        anchors.fill: parent
-        opacity: 0
-        visible: opacity > 0.001
-        color: Theme.palette.baseColor2
-    }
-
-    MouseArea {
-        id: mouseArea
-        anchors.fill: parent
-    }
+        Rectangle {
+            id: highlightRect
+            anchors.fill: parent
+            opacity: 0
+            visible: opacity > 0.001
+            color: Theme.palette.baseColor2
+        }
 
-    HoverHandler {
-        id: hoverHandler
-        enabled: !root.isActiveMessage && !root.disableHover
-    }
+        MouseArea {
+            id: mouseArea
+            anchors.fill: parent
+        }
 
-    ColumnLayout {
-        id: messageLayout
-        anchors.fill: parent
-        anchors.topMargin: 2
-        anchors.bottomMargin: 2
-
-        Loader {
-            Layout.fillWidth: true
-            active: isAReply
-            visible: active
-            sourceComponent: StatusMessageReply {
-                replyDetails: root.replyDetails
-                profileClickable: root.profileClickable
-                onReplyProfileClicked: root.replyProfileClicked(sender, mouse)
-                audioMessageInfoText: root.audioMessageInfoText
-                onLinkActivated: {
-                    root.linkActivated(link);
-                }
-            }
+        HoverHandler {
```

Comment:
This `HoverHandler` is kinda superfluous now... your toplevel item is a `Control` now which already has this functionality :)

Reply:
done